### DISCUSSION TITLE
PreCommit1 to PreCommit2 help

### DISCUSSION THREAD

**Discussion by lioncrowe** [filecoin-project/lotus#7771] [Storage Provider]
I am a beginner. It is currently running until lotus-miner run.
However, it does not move from PreCommit1 to PreCommit2 in the lotus-miner sectors list.
What should I do?

**Comment by Jhin1128** [comment]
Same question.Can you solve this question?

**Comment by Reiers** [comment]
Moving this to discussions for troubleshooting and help.

What lotus version are you on? and please share your logs.
